nixos/modules/config/fonts/packages.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.fonts;
+in
+{
+  imports = [
+    (lib.mkRemovedOptionModule [ "fonts" "enableCoreFonts" ] "Use fonts.packages = [ pkgs.corefonts ]; instead.")
+    (lib.mkRenamedOptionModule [ "fonts" "enableDefaultFonts " ] [ "fonts" "enableDefaultPackages" ])
```

Comment:
There is an extra space here :(

Reply:
:sob:

Reply:
Waiting the fix to enableDefaultPackages :3

Reply:
@linuxmobile https://github.com/NixOS/nixpkgs/pull/245380 or do you mean something else?